File: Iniciante/1074.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

/*
 * 
*/
public class Main {
	public static void main(String[] args) throws IOException {
		
		Scanner sc = new Scanner(System.in);
		Locale.setDefault(Locale.US);
		
		int number_cases,count_in=0,count_out=0,n;
		number_cases=sc.nextInt();
		
		for(int i=0;i<number_cases;i++) {
			n=sc.nextInt();
			
			if(n<0 && n%2==0) {
				System.out.println("EVEN NEGATIVE");
			}
			if(n>0 && n%2==0) {
				System.out.println("EVEN POSITIVE");
			}
			if(n<0 && n%2!=0) {
				System.out.println("ODD NEGATIVE");
			}
			if(n>0 && n%2!=0) {
				System.out.println("ODD POSITIVE");
			}
			if(n==0) {
				System.out.println("NULL");
			}
		}
		
		
		
		sc.close();
	}

}

File: Beginner/1074.py

numberCases = int(input())

for i in range(0, numberCases, 1):
    n = int(input())
    if n < 0 and n % 2 == 0:
        print("EVEN NEGATIVE")
    elif n > 0 and n % 2 == 0:
        print("EVEN POSITIVE")
    elif n < 0 and n % 2 != 0:
        print("ODD NEGATIVE")
    elif n > 0 and n % 2 != 0:
        print("ODD POSITIVE")
    elif n == 0:
        print("NULL")